### ISSUE TITLE
Feature: Nav Slide In

### ISSUE THREAD

**Issue by jonrutter** [jonrutter/demo-photo-portfolio#3]
Make nav dropdown slide in and out on opening.

**Comment by jonrutter** [comment]
Make sure to maintain accessibility. Doing this will likely make it impossible to keep `display: none;` on the nav dropdown when closed. Make sure that users cannot tab to the links when hidden.

**Comment by jonrutter** [comment]
Finished.
